frontends/web/src/routes/device/bitbox02/passphrase.tsx

```diff
@@ -35,345 +36,360 @@ const INFO_STEPS_DISABLE = 0;
           verticallyCentered>
-          <ViewHeader title={t('passphrase.summary.title')} />
+          <ViewHeader
+            small
+            title={t(passphraseEnabled
+              ? 'passphrase.successDisabled.title'
+              : 'passphrase.successEnabled.title')} >
+          </ViewHeader>
           <ViewContent>
-            <ul>
-              <SimpleMarkup key="info-1" tagName="li" markup={t('passphrase.summary.understandList.0')} />
-              <SimpleMarkup key="info-2" tagName="li" markup={t('passphrase.summary.understandList.1')} />
-              <SimpleMarkup key="info-3" tagName="li" markup={t('passphrase.summary.understandList.2')} />
-              <SimpleMarkup key="info-4" tagName="li" markup={t('passphrase.summary.understandList.3')} />
-            </ul>
-            <Status type={understood ? 'success' : 'warning'}>
-              <Checkbox
-                onChange={e => this.setState({ understood: e.target.checked })}
-                id="understood"
-                checked={understood}
-                label={t('passphrase.summary.understand')}
-                checkboxStyle={understood ? 'success' : 'warning'} />
-            </Status>
+            <MultilineMarkup tagName="p" markup={t(passphraseEnabled
+              ? 'passphrase.successDisabled.message'
+              : 'passphrase.successEnabled.message')} />
+            {passphraseEnabled && (
+              <ul style={{ paddingLeft: 'var(--space-default)' }}>
+                <SimpleMarkup key="tip-1" tagName="li" markup={t('passphrase.successEnabled.tipsList.0')} />
+                <SimpleMarkup key="tip-2" tagName="li" markup={t('passphrase.successEnabled.tipsList.1')} />
+              </ul>
+            )}
+            <SimpleMarkup tagName="p" markup={t(
+              passphraseEnabled
+                ? 'passphrase.successDisabled.messageEnd'
+                : 'passphrase.successEnabled.messageEnd')} />
           </ViewContent>
-          <ViewButtons>
-            <Button primary onClick={this.continueInfo} disabled={!understood}>
-              {t('passphrase.enable')}
-            </Button>
-            <Button secondary onClick={this.backInfo}>
-              {t('button.back')}
-            </Button>
-          </ViewButtons>
         </View>
-      );
-    default:
-      console.error(`invalid infoStep ${infoStep}`);
-      return;
-    }
-  };
+      )}
+    </Main>
+  );
+};
+
+type TEnableInfoProps = {
+  onClose: () => void;
+  onContinue: () => void;
+  onBack: () => void;
+  infoStep: number;
+  understood: boolean;
+  // will probably remove this anyways so use any for now.
+  setUnderstood: any;
+}
 
-  private renderDisableInfo = () => {
-    const { t } = this.props;
+type TDisableInfoProps = {
+  onClose: () => void;
+  onContinue: () => void;
+  onBack: () => void;
+}
+
+const EnableInfo = ({ onClose, onContinue, onBack, infoStep, understood, setUnderstood }: TEnableInfoProps) => {
+  // Shoud t be passed as a prop?
```

Comment:
`// Shoud t be passed as a prop?`

Do you mean the next line? keep `const { t } = useTranslation();` don't pass as prop. if that was you question.